templates/education/dnso-profiles.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+//**
+ //* A template for DNSO Profiles.  http://dnso.educ.ubc.ca/administrative-directory/
+
+// Fetch the post ID for the currently set up post - we're in the loop
+
+$post_id      = get_the_ID();
+
+// How to get the post thumbnail
+$post_thumbnail   = wp_get_attachment_url( get_post_thumbnail_id( absint( $post_id ) ) );
+
+// Replacement for [plain_tags_slug]
+$plain_tags_slug  = \UBC\Helpers::get_plain_tags( absint( $post_id ), 'slug', ' ' );
+
+// Replacement for [plain_term_slug]
+$plain_term_slug  = \UBC\Helpers::get_plain_terms( absint( $post_id ), 'slug', ' ' );
+
+// To get Taxonomies as names
+$plain_term_name  = \UBC\Helpers::get_plain_terms( absint( $post_id ), 'name', ' ' );
+
+
+// Replacement for [odd-even] but *really* this should be done in CSS.
+global $usage_id;
+$usage_id++;
+$odd_even       = ( 0 === $usage_id % 2 ) ? 'even' : 'odd';
+
+//Get the Profile Fields 
+$profile_cct  = get_post_meta( $post_id, 'profile_cct',true);
+
+$salutations         = ($profile_cct['name']['salutations']) ;
```

Comment:
Encompassing each of these variables in parentheses is ... weird. There's a couple of things to note;

1. (picky) There needs to be spaces after/before opening and closing parentheses.
2. If you're trying to do a ternary operator here (which is normally why you use parentheses like this) then you need to be more specific, i.e.

`$first_name          = ( isset( $profile_cct['name']['first'] ) ) ? $profile_cct['name']['first'] : false;`

This would mean that the value of `$first_name` would be equal to the value of `$profile_cct['name']['first']` if it's set and `false` if it isn't. Then, whenever you come to use `$first_name` you can check if it's set to `false` and if it isn't, use it. i.e.

`if ( false !== $first_name ) : do_stuff_with_first_name(); endif;` or something like that